### ISSUE TITLE
Dead spot on health log demographic tap targets

### ISSUE THREAD

**Issue by dharding** [covid-alert-ny/covid-green-app#256]
On the form where we collect the optional demo data, there are two dead spots that taps do not work on. I have circled their approximate locations in the screenshot.
![Screenshot_20200902-180453](https://user-images.githubusercontent.com/8274/92041956-396b0f80-ed47-11ea-8ea8-ab80f02bf952.png)

**Comment by floridemai** [comment]
@dharding This is how is implemented: each item has a padding and the right side is the padding on the card/itself. @helloantoine Please advise on wether all of the area between the separator should be actionable. What about the right side padding? (do we need to occupy the full width and position the arrow inside; now the arrow is the last  element)

**Comment by davidjharding** [comment]
All of the area between the horizontal separators should be an active tap target.

On the right side, what I was experiencing was that the area right near the right arrow icon was very specific about where I had to tap.

I think it makes sense to make the entire area vertically between the separators and the entire area that spans the width of the separators plus some slop to account for inaccuracy (See https://reactnative.dev/docs/view.html#hitslop) as the tap target. Basically, the entire parent view should be the tap target.

**Comment by JohnAGill** [comment]
This has been addressed and will be in the next build

**Comment by helloantoine** [comment]
Looks good to me. Good job!
